packages/types/mongoose-store/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as Promise from 'bluebird';
+import * as mongoose from 'mongoose';
+
+export default class MongooseStore<T>{
+  constructor(protected conn: mongoose.Connection) {
+
+  }
+}
+
+type ID = string | number;
```

Comment:
Can we use `any` for id as this may provide some blockers for users who will want to use for example Objectid